pyproject.toml

```diff
@@ -10,7 +10,7 @@ include = [
 ]
```

Comment:
let's update the `version` above to a minor semver update. So 3.2.0 and then we can do a service release after this